### PR TITLE
Emailable interface

### PR DESCRIPTION
**Pull request by gymad**
## Description

Emailable interface

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/Contacts/Contact.php [hunk 1]
```diff
@@ -186,6 +187,11 @@ public function Contact()
         }
         self::__construct();
     }
+    
+        
+    public function getEmailBean() {
```
Comment (PedroErnst) [inline modules/Contacts/Contact.php]: The PHPDocs seem to be missing

File: modules/Leads/Lead.php [hunk 2]
```diff
@@ -164,6 +165,12 @@ public function Lead(){
         }
         self::__construct();
     }
+    
+    
+        
+    public function getEmailBean() {
```
Comment (PedroErnst) [inline modules/Leads/Lead.php]: The PHPDocs seem to be missing

File: modules/Prospects/Prospect.php [hunk 3]
```diff
@@ -124,6 +125,11 @@ function Prospect(){
         }
         self::__construct();
     }
+    
+    
+    public function getEmailBean() {
```
Comment (PedroErnst) [inline modules/Prospects/Prospect.php]: The PHPDocs seem to be missing

File: modules/Users/User.php [hunk 4]
```diff
@@ -147,6 +148,11 @@ public function User()
         }
         self::__construct();
     }
+       
+    public function getEmailBean() {
```
Comment (PedroErnst) [inline modules/Users/User.php]: The PHPDocs seem to be missing

File: modules/Contacts/Contact.php (hunk 1 above)
Comment (gymad) [reply modules/Contacts/Contact.php]: in the interface but I can add it here also

**Pull request closed without merging**